webapp/src/js/services/feedback.js

```diff
@@ -41,7 +43,10 @@ angular.module('inboxServices').factory('Feedback',
 
     var create = function(info, isManual, callback) {
       var userCtx = Session.userCtx();
+      let currentDate = new Date().toISOString().split('T')[0];
```

Comment:
It may be useful to have the full datetime, so just drop the split. That way two feedbacks on the same day will still be ordered correctly.

Also use a `const` wherever possible, and `let` only you are reassigning the variable.

Finally use the same date in the meta rather than generating again, eg:

```js
const date = new Date().toISOString();
const uuid = uuidV4();
callback(null, {
  _id: `feedback-${date}-${uuid}`,
  meta: {
    time: date,
    ...
```